src/index.html

```diff
@@ -1,13 +1,359 @@
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="fonts">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header 
+      class="header"
+    >
+      <a href="#home" class="header__logo">
+        <img
+          src="images/logocopy.png"
+          alt="logo"
+          class="header__logo-image"
+        >
+      </a>
+      <nav 
+        class="header__nav"
+      >
+        <ul class="header__nav-list">
+          <li class="header__nav-card">
+            <a
+              href="#Apple"
+              class="header__nav-link header__nav-link--is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-card">
+            <a 
+              href="#Samsung" 
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__nav-card">
+            <a 
+              href="#Smartphones" 
+              class="header__nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="header__nav-card">
+            <a
+              href="#Laptops&Computers"
+              class="header__nav-link"
+             
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__nav-card">
+            <a 
+              href="#Gadgets" 
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__nav-card">
+            <a 
+              href="#Tablets" 
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__nav-card">
+            <a 
+              href="#Photo" 
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__nav-card">
+            <a 
+              href="#Video" 
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="container">
+        <li class="card">
+          <p class="card__item card__item--img">
+            <a href="#" class="card__img">
+              <img 
+              src="./images/imac.jpeg" 
+              alt="mac"
+              class="card__picture"
+            >
```

Comment:
```suggestion
              <img 
                src="./images/imac.jpeg" 
                alt="mac"
                class="card__picture"
              >
```
please pay attention to formatting